File: chapter6/conditions.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    // declare variable for percent
    int percent;
    
    // prompt the user for input
    printf("Enter a valid grade: ");
    
    // use GetInt to get user input and store in variable
    percent = GetInt();
    
    // if percent is 90 or above, print A
    if (percent >= 90)
    {
        printf("You got an A!\n");
    }
    else if (percent >= 80)
    {
        printf("You got a B!\n");
    }
    else if (percent >= 70)
    {
        printf("You got a C!\n");
    }
    else if (percent >= 0)
    {
        printf("You got a D!\n");
    }
    else
    {
        printf("You somehow got a negative grade. Sad.\n");
    }
}

File: chapter6/conditions.py
import cs50

# method one: CS50 library
print("Enter a valid grade: ", end="")
percent = cs50.get_int()

# method two: Python input
# percent = input("Enter a valid grade: ")
# input is always a string; need this line to convert to int
# percent = int(percent)

if percent >= 90:
    print("You got an A!")
elif percent >= 80:
    print("You got a B!")
elif percent >= 70:
    print("You got a C!")
elif percent >= 0:
    print("You got a D!")
else:
    print("You somehow got a negative grade. Sad.")